src/main/java/com/cst438/controller/EnrollmentController.java

```diff
@@ -14,29 +18,57 @@
 @CrossOrigin(origins = "http://localhost:3000")
 public class EnrollmentController {
 
+    @Autowired
+    EnrollmentRepository enrollmentRepository;
+
     // instructor downloads student enrollments for a section, ordered by student name
     // user must be instructor for the section
     @GetMapping("/sections/{sectionNo}/enrollments")
+    @PreAuthorize("hasAuthority('SCOPE_ROLE_INSTRUCTOR')")
     public List<EnrollmentDTO> getEnrollments(
             @PathVariable("sectionNo") int sectionNo ) {
 
```

Comment:
I think we should pass in the Principal argument here so we can check that the caller of the method is actually the instructor of the section. See ```getAssignments``` method in assignmentController for a similar example.